libs/sdk-ui-tests-e2e/cypress/tools/attributeFilterConfig.ts

```diff
@@ -15,6 +17,11 @@ export class AttributeFilterConfiguration {
         return this;
     }
 
+    selectDisplayValueButton() {
+        this.clickElement(ATTR_DISPLAY_FORM_BUTTON);
```

Comment:
use click() 
cy.get...click()
why do we use clickElement() although we already have click()?